test/e2e/snaps/test-snap-multi-install.spec.js

```diff
@@ -47,6 +47,10 @@ describe('Test Snap Multi Install', function () {
         });
 
         // wait and scroll if necessary
+        await driver.waitForSelector({
+          tag: 'h3',
+          text: 'Add to MetaMask',
+        });
```

Comment:
since in some tests we don't use page objects, we need to add this individually in those tests